src/screens/DiscoveryScreenV2/index.js

```diff
@@ -65,10 +66,6 @@ const DiscoveryScreenV2 = ({route}) => {
 
   const navigation = useNavigation();
 
-  const handleScroll = React.useCallback(() => {
-    Keyboard.dismiss();
-  });
-
   React.useEffect(() => {
     const unsubscribe = () => {
       DiscoveryAction.setDiscoveryFirstTimeOpen(true, discoveryDispatch);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `handleScroll` function, which was responsible for dismissing the keyboard, might have implications on the user experience if there are no other mechanisms in place to dismiss the keyboard when scrolling. Please ensure that this functionality is handled elsewhere.
```diff
-  const handleScroll = React.useCallback(() => {
-    Keyboard.dismiss();
-  });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->